starter-code/routes/celebrities.js

```diff
@@ -0,0 +1,57 @@
+
+const celibrityModel = require("./../models/Celebrity");
+
+module.exports = function(router) {
+
+  // GET all celebrities route page
+  router.get("/celebrities", (req, res) => {
+    celibrityModel
+      .find() //
+      .then(dbResults => {
+        res.render("celebrities/index", {
+          celebrities: dbResults
+        });
+      })
+      .catch(dbErr => {
+        console.log("OH NO ! Database error", dbErr);
+      });
+  });
+
+  router.post("/celebrities", (req, res) => {
+    const { name, occupation, catchPhrase } = req.body;
+ 
+    celibrityModel
+      .create({ name, occupation, catchPhrase })
+      .then(dbRes => res.redirect("/celebrities"))
+      .catch(dbErr => {
+        console.log(dbErr);
+        res.render("celebrities/new") 
+      });
+    // res.render("celebrities/new") 
+  })
+
+  router.get("/celebrities/new", (req, res) => {
+    res.render("celebrities/new") 
+  })
+
+  router.get("/celebrities/:id", (req, res) => {
+    celibrityModel
+      .findById(req.params.id)
+      .then(dbRes => {
+        res.render("celebrities/show", { celebrityDetails: dbRes });
+      })
+      .catch(dbErr => console.error(dbErr));
+  });
+
+  
+  router.post("/celebrities/:id/delete", (req, res) => {
+ 
+    celibrityModel
+      .findByIdAndDelete(req.params.id)
+      .then(dbRes => {
+        res.redirect("/celebrities");
+      })
+      .catch(dbErr => console.error(dbErr));
+  })
+
+};
```

Comment:
It would be better to create a router and export that router.

When you use your router in "app.use(yourRouter)" you are able to either prefix the router, or set a middleware for the 
a specific router if needed, you would need to pass in more arguments to your function in order to achieve the same thing.